src/components/SettingsDialog/ExportBoard/ExportHintHiddenColumns/ExportHintHiddenColumns.scss

```diff
@@ -0,0 +1,12 @@
+@import "src/constants/style.scss";
+
+.export-board__hint-hidden-columns {
+  margin-top: $margin--default;
+  padding: 0 $padding--default 0 $padding--default;
+}
+
+[theme="dark"] {
+  .export-board__hint-hidden-columns {
+    color: $color-white;
+  }
+}
```

Comment:
Seems a bit bland to me, but if you dont want to add more, this will be good enough.

Reply:
yea, it's quite basic I know ;) 
I'm gonna go ahead and implement @Dominik-Weinzierl 's suggestion: https://github.com/inovex/scrumlr.io/pull/2135#issuecomment-1238121766

Reply:
You can look into the keyboard shortcut hint 
![image](https://user-images.githubusercontent.com/44020029/189049502-ddfae02b-e9bc-4e8a-8121-82830a88cca7.png)
![image](https://user-images.githubusercontent.com/44020029/189049524-a2ac934e-1283-4f1a-a51f-cf173f198236.png)

But I agree with  @Dominik-Weinzierl a exclamation mark icon or some highlighting would be great

Reply:
Looks good to me - can you reduce the spacing between the bullet points and the first text?

Reply:
> I adjusted the style a bit, what do you think?  
> colorful text was a bit much, the icon is enough for that purpose imo
> 
> ![export-hint-v2](https://user-images.githubusercontent.com/7889564/189125226-cd3a448a-4b74-421b-bbd8-bf50493b85ca.png)
> 

Could you also set the icon width and height? There are some predefined variables, pick a size you like (e.g. `$icon--medium`)
Thank you very much

Reply:
@CronJorian that's already the case :) although the size isn't consistent on different viewport sizes for some reason

Reply:
@Dominik-Weinzierl i added it (12e927b201e920e03313f8bb53e3cfed3f4d4f5f), however I'm not completely happy with the solution.  
the viewport changes size at `920px`, which is still hardcoded into `SettingsDialog.scss`. This viewport change leads to the text being split up into three lines instead of two, requiring a different margin, hence the media query. I guess I could outsource the `920px` to `style.scss`?

Reply:
There are already breakpoints defined in the style.scss, you can search for `#smartphone` to find them